src/index.js

```diff
@@ -1,36 +1,38 @@
-import {parse as parseCookies} from 'cookie';
-import httpsModule from 'https';
-import {getPEM} from 'pan-domain-public-keys';
-import validateUser from 'pan-domain-validate-user';
-import {STAGE} from './environment';
+import {guardianValidation, PanDomainAuthentication} from '@guardian/pan-domain-node'; 
+import {SETTINGS_FILE, REGION} from './environment';
 
 export function handler (events, context, callback) {
-	handleEvents({events, callback, https: httpsModule, logger: console, validate: validateUser});
+	const panda = new PanDomainAuthentication('gutoolsAuth-assym', REGION, 'pan-domain-auth-settings', SETTINGS_FILE, guardianValidation);
+	handleEvents({events, callback, panda, logger: console});
 }
 
-export function handleEvents ({events, callback, https, logger, validate}) {
-	const cookie = getPandaCookie(events.authorizationToken || '');
+export function handleEvents ({events, callback, panda, logger}) {
+	const cookie = events.authorizationToken || '';
 
-	getPEM(STAGE, https)
-	.then(key => validate(cookie, key))
-	.then(user => {
-		callback(null, policy(
-			`${user.firstName} ${user.lastName} <${user.email}>`,
-			'Allow',
-			events.methodArn
-		));
+	panda.verify(cookie)
+	.then(({ status, user }) => {
+		if (status === 'Authorised') {
```

Comment:
`status` doesn't match numerical HTTP Status codes? 😢

Reply:
Maybe not the best choice of variable names but it is at least a [Typescript enum](https://github.com/guardian/pan-domain-authentication/blob/master/pan-domain-node/src/api.ts#L3) in the API, the idea being you can then map that on to framework you're using. For example I don't think you can control the status code returned from the custom authorizer (maybe?), instead just returning `Allow` or `Deny` to the callback